src/net/net.c

```diff
@@ -149,7 +149,7 @@ static boolean get_static_config(tuple root, struct netif *n, boolean trace) {
 }
 
 void init_network_iface(tuple root) {
-    struct netif *n = netif_find("en0");
+    struct netif *n = netif_find("en1");
```

Comment:
It doesn't matter much at this point, but could this be renamed to lo0, just to differentiate it from additional ethernet interfaces we might want to add in the future?

Reply:
Correction: I meant to ask if this could stay as en0. Isn't the loopback interface already lo0? (see netif_loopif_init() in lwip/src/core/netif.c)

Reply:
Interface names in lwIP are composed of a two-letter identifier followed by a number, and the number corresponds to the 0-based index of the interface in the global interface list. The loopback interface, if present, is always added first (it is added from `netif_init()`), so the next interface must have index number 1 and so if its identifier is "en" it must be called "en1". Any additional ethernet interfaces we might add in the future will be named, according to the order in which they are added, "en2", "en3", etc.